src/util/rethrow.js

```diff
@@ -46,7 +46,7 @@ const rethrowAndCountFspiopError = (error, options = {}, context) => {
 const countFspiopError = (error, options = {}, context) => {
   const { operation, step, loggerOverride } = options
   const log = loggerOverride || logger
-  if (error?.message) log.error(error?.message, error)
+  if (error?.message) log.warn('countFspiopError: ', error)
```

Comment:
The code that was refactored to use this function was logging at error level, so we need to keep it.
I also suggest that we put the operation and step in the prefix, for example:
`countFspiopError(${operation}${step ? '.' + step : ''}): `